src/rqt_console/message.py

```diff
@@ -125,7 +128,11 @@ def pretty_print(self):
         text += self.tr('Time: ') + self.get_stamp_string() + '\n'
         text += self.tr('Severity: ') + Message.SEVERITY_LABELS[self.severity] + '\n'
         text += self.tr('Published Topics: ') + ', '.join(self.topics) + '\n'
-        text += '\n' + self.message.decode('utf-8') + '\n'
+        # Python version is checked here in order to solve 
+        # https://github.com/ros-visualization/rqt_console/issues/33
+        # Python 2 requres the data to be decoded to avoid crash when the string contains
```

Comment:
Great, thanks for the comment I asked for.
Please just correct the typo in required and I can merge your PR. :)

Reply:
Apologies for the delay, I fixed the typo